src/ApiService/ApiService/onefuzzlib/NodeOperations.cs

```diff
@@ -497,17 +507,6 @@ public NodeOperations(ILogTracer log, IOnefuzzContext context)
             queryParts.Add($"({q})");
         }
 
-        if (excludeUpdateScheduled) {
```

Comment:
this was first bug - SearchStatesQuery would ignore current version node. We do not want that. (And Python code does not ignore it ether, so this fix is just aligning implementation with Python code)